packages/ui/src/services/global-config/global-config.ts

```diff
@@ -42,30 +42,19 @@ export const createGlobalConfig = () => {
   }
 }
 
+/** Use this function if you don't want to throw error if hook used ouside setup function by useGlobalConfig */
+export function useGlobalConfigSafe () {
+  return inject<ProvidedGlobalConfig>(GLOBAL_CONFIG)
+}
+
 export function useGlobalConfig () {
-  const globalConfig = inject<ProvidedGlobalConfig>(
-    GLOBAL_CONFIG,
-    getCurrentInstance()?.appContext.config.globalProperties.$vaConfig,
-  )
+  const globalConfig = inject<ProvidedGlobalConfig>(GLOBAL_CONFIG)
```

Comment:
Let's rename this variable (e.g. `globalConfigInstance`) because `ProvidedGlobalConfig` also contains `globalConfig`